pkg/hwevent/event_data.go

```diff
@@ -168,22 +169,22 @@ type RedfishEvent struct {
 func (e RedfishEvent) String() string {
 	b := strings.Builder{}
 	if e.OdataContext != "" {
-		b.WriteString("\n@odata.context: " + e.OdataContext + "\n")
+		b.WriteString("\n    @odata.context: " + e.OdataContext + "\n")
 	}
-	b.WriteString("@odata.type: " + e.OdataType + "\n")
+	b.WriteString("    @odata.type: " + e.OdataType + "\n")
 	if e.Actions != nil {
-		b.WriteString("Actions: " + string(e.Actions) + "\n")
+		b.WriteString("    Actions: " + string(e.Actions) + "\n")
 	}
 	if e.Context != "" {
-		b.WriteString("Context: " + e.Context + "\n")
+		b.WriteString("    Context: " + e.Context + "\n")
 	}
-	b.WriteString("Id: " + e.ID + "\n")
-	b.WriteString("Name: " + e.Name + "\n")
+	b.WriteString("    Id: " + e.ID + "\n")
+	b.WriteString("    Name: " + e.Name + "\n")
 	if e.Oem != nil {
-		b.WriteString("Oem: " + string(e.Oem) + "\n")
+		b.WriteString("    Oem: " + string(e.Oem) + "\n")
 	}
 	for i, e := range e.Events {
-		b.WriteString("Events[" + strconv.Itoa(i) + "]:\n")
+		b.WriteString("    Events[" + strconv.Itoa(i) + "]:")
 		b.WriteString(e.String())
 	}
 	return b.String()
```

Comment:
everything is using tab

Reply:
This is 4 spaces. Same reason as the last comment.